src/main/java/ch/njol/skript/expressions/ExprWorld.java

```diff
@@ -53,14 +56,15 @@
 @Since("1.0")
 public class ExprWorld extends PropertyExpression<Object, World> {
 	static {
-		Skript.registerExpression(ExprWorld.class, World.class, ExpressionType.PROPERTY, "[the] world [of %location/entity%]", "%location/entity%'[s] world");
+		Skript.registerExpression(ExprWorld.class, World.class, ExpressionType.PROPERTY, "[the] world [(of|from) %location/entity%]", "%location/entity%['s] world");
```

Comment:
I'd like this be changed to:
```java
[the] world[s] [of %locations/entities%]
%locations/entities%'[s] world[s]
```
* It uses the plural form of the objects because it's improves the usefulness of the expression and property expressions normally accept multiple objects. Doesn't look like it's gonna break something looking at the class; if anything, I'll test it after changes are made.
* Removed the `from` and made the apostrophe obligatory again because whether it's proper grammar or not, the same way as above, I'd like to keep a bit of consistency. If @bensku doesn't mind this one you can keep it that way though.